### PR TITLE
Fixed a few syntax errors: Caleb

### LINKED ISSUE #126: Setup RDS
As a DevOps engineer, I want to set up the RDS (Relational Database Service) with the required database, tables, and seeded data, so that the application has a cloud-based, scalable, and secure backend to store and retrieve stargazing data for the users.

Setup the RDS:
- Get credentials
- Run Schema
- Seed with static data
- Seed with test data

### LINKED ISSUE #132: Fix Syntax Error in Test Data
As a developer, I want to fix the test data for the stargazing database, so that accurate and representative data can be used for testing the pipeline and features, ensuring the application behaves as expected under different conditions and datasets.

Fix Syntax Errors:
- Missing Commas
- Missing Semi-colons

### PR DESCRIPTION
**Pull request by CalebMathers**
This closes #132 
This closes #126 

Just fixed a few missing commas and a semi-colon. Also changed one of the region ids in image insert as it was out of bounds.

RDS is all set up too with the tables, static data, and some test data.

### CODE REVIEW

File: database/fake_data.sql [hunk 1]
```diff
@@ -79,7 +79,7 @@ VALUES
 ('davidjohnson', '+12345678907', 'davidjohnson@example.com'),
 ('sarahwilson', '+12345678908', 'sarahwilson@example.com'),
 ('chrislee', '+12345678909', 'chrislee@example.com'),
-('lauragarcia', '+12345678910', 'lauragarcia@example.com')
+('lauragarcia', '+12345678910', 'lauragarcia@example.com'),
```
Comment (AlexOBarnes) [inline database/fake_data.sql:82]: Good catch Caleb

File: database/fake_data.sql [hunk 2]
```diff
@@ -103,7 +103,7 @@ VALUES
 (89, 4),
 (19, 10),
 (76, 2),
-(41, 13)
+(41, 13),
```
Comment (AlexOBarnes) [inline database/fake_data.sql:106]: I didn't spot this very good catch

**Review by AlexOBarnes — APPROVED** [review]
Thanks for these changes

File: database/fake_data.sql (hunk 2 above)
Comment (MurasatoNaoya) [reply database/fake_data.sql:106]: LGTM. These kind of things slip through unless we run it on an example database, good update.

**Review by MurasatoNaoya — APPROVED** [review]

File: database/fake_data.sql (hunk 2 above)
Comment (AlexOBarnes) [reply database/fake_data.sql:106]: Great learning opportunity for the deputy quality assurer

**Review by olliejth — APPROVED** [review]
LGTM

**Pull request merged**